src/main/play-doc/operation/Install.md

```diff
@@ -729,6 +784,31 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
 
 You are now ready to [[deploy bundles|DeployingBundlesOps]] into ConductR.
 
+
+## Loading and Running ConductR-HAProxy Bundle
+
+ConductR-HAProxy bundle will expose services provided by the bundle so it can be accessed from outside cluster. ConductR-HAProxy will ensure the HAProxy configuration is kept update based on the bundles which are running.
+
+Obtain the ConductR HAProxy bundle from the ConductR installation.
+
+* Open the Mesos Web Console
+* Locate the ConductR Core task running in the list of Active Tasks. The task is called `conductr-{version}` or something similar.
+* Click the Sandbox link to view the filesystem of the deployed ConductR Core.
+* Navigate to the directory `conductr-{version}-{digest}/extra`.
+* Locate the ConductR HAProxy bundle in a file called `conductr-haproxy-v2-{digest}.zip`.
+* Download the file.
```

Comment:
Why not just download the ConductR distribution the bastin host and then distribute the file from there?